scripts/tasks/chromatic.ts

```diff
@@ -15,6 +15,9 @@ export const chromatic: Task = {
     await exec(
       `npx chromatic \
           --exit-zero-on-changes \
+          --exit-once-uploaded \
+          --only-changed \
+          --debug \
```

Comment:
This is only necessary if we want to enable TS for the sandbox SBs, which I don't believe is the aim of this PR. The `ui:` SBs already have `--only-changed` turned on it seems: https://github.com/storybookjs/storybook/blob/737d3399272343a76a1cd17832af70671ea4c26c/code/package.json#L84